test/NuGet.Core.FuncTests/Dotnet.Integration.Test/PackCommandTests.cs

```diff
@@ -5711,5 +5711,76 @@ public void Dotnet_New_Template_Restore_Pack_Success(string template)
                 Assert.True(File.Exists(nupkgPath));
             }
         }
+
+
+        [Theory]
+        [InlineData("net472")]
+        [InlineData("net5.0")]
+        public void PackCommand_PackProject_PackageReference_PreReleaseDependency(string platform)
+        {
+            // Arrange
+            using (var testDirectory = msbuildFixture.CreateTestDirectory())
```

Comment:
You should use SimpleTestContext, which creates a global packages folder, local file feed source, so it's easy to use generated test packages, rather than packages from nuget.org, and it won't pollute the global packages folder of anyone who runs the test locally.